plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -206,7 +207,37 @@ public static void rootConfigGeneration() throws Exception {
             final Mapping mapping = config.asMapping();
             final List<Map.Entry<String, CNode>> entries = new ArrayList<>(mapping.entrySet());
             for (Map.Entry<String, CNode> entry : entries) {
-                System.out.println(entry.getKey() + " " + entry.getValue().toString());
+                System.out.println(entry.getKey());
+            }
+            System.out.println("End of an iteration of configurators");
+        }
+    }
+
+    public static void storeConfiguratorNames() {
+        ConfigurationAsCode configurationAsCodeObject = ConfigurationAsCode.get();
+        for (Object configuratorObject : configurationAsCodeObject.getConfigurators()) {
+            if (configuratorObject instanceof BaseConfigurator) {
+                BaseConfigurator baseConfigurator = (BaseConfigurator) configuratorObject;
+                List<Attribute> baseConfigAttributeList = baseConfigurator.getAttributes();
+                for (Attribute attribute : baseConfigAttributeList) {
+                    if (attribute.multiple) {
+                        System.out.println(
+                            "This is a multiple attribute " + attribute.getType() + " " + attribute
```

Comment:
@timja We need a way to get the most out of the multiple attributes.
`This is a multiple attribute class hudson.slaves.NodeProperty globalNodeProperties`
If we could expand globalNodeProperties that would be great.